media-video/obs-studio/metadata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>qt@gentoo.org</email>
+    <name>Gentoo Qt Project</name>
+  </maintainer>
+  <longdescription>
+    A rewrite of what was formerly known as "Open Broadcaster Software",
+    software originally designed for recording and streaming live video
+    content, efficiently.  Aims to be multi-platform, and improved upon
+    internally.
+  </longdescription>
+  <use>
+    <flag name="alsa">Enable the use of ALSA for audio recording.</flag>
+    <flag name="fdk">Enable libfdk support for AAC encoding.</flag>
+    <flag name="imagemagick">Use ImageMagick for image loading instead of FFmpeg.</flag>
+    <flag name="jack">Add support for the JACK Audio Connection Kit.</flag>
+    <flag name="pulseaudio">Enable the use of PulseAudio for audio recording.</flag>
+    <flag name="truetype">Enable the Freetype text plugin.</flag>
+    <flag name="v4l">Enable the v4l2 plugin.</flag>
```

Comment:
I'd remove all local USE descriptions except for `fdk` and `imagemagick`, since they're all very similar to the global descriptions.